content/posts/July-2020.md

```diff
@@ -18,3 +18,18 @@ draft: false
 * You are now able to override configuration for builds, releases and other jobs (by [@TomasTomecek](https://github.com/TomasTomecek), [#676](https://github.com/packit-service/packit-service/pull/676)).
  Read more about this feature [here](/docs/configuration/#overriding-global-parameters).
 
+
+## Week 28 (July 6th - July 10th)
+
+### packit
+
+* Command `validate-config` is now working properly and not producing odd error messages, by [@TomasTomecek](https://github.com/TomasTomecek) and [@csomh](https://github.com/csomh), [#894](https://github.com/packit-service/packit/pull/894) and [#895](https://github.com/packit-service/packit/pull/895)
```

Comment:
I'd use simple tense since the continuous suggests that it is working now, but might stop working at some point :-)
```suggestion
* Command `validate-config` works properly now and does not produce odd error messages, by [@TomasTomecek](https://github.com/TomasTomecek) and [@csomh](https://github.com/csomh), [#894](https://github.com/packit-service/packit/pull/894) and [#895](https://github.com/packit-service/packit/pull/895)
```